### PR TITLE
Fetch states, districts, Taluks, Villages 4 apis (WO Auth)

### PR DESCRIPTION
**Pull request by babstar93**
Get Village list by district ID api

### CODE REVIEW

File: .github/workflows/sast-and-package-prod.yml [hunk 1]
```diff
@@ -30,7 +32,7 @@ jobs:
         languages: Java    
         
     - name: Build with Maven
-      run: mvn clean install -Denvironment=prod
+      run: mvn clean install -DENV_VAR=${{ env.ENV_VAR }}
```
Comment (roopesh-beehyv) [inline .github/workflows/sast-and-package-prod.yml]: We are not supposed to change other files unless there is a bug or strong use case. Revert this change

File: src/main/java/com/iemr/hwc/controller/location/LocationController.java [hunk 2]
```diff
@@ -66,7 +66,7 @@ public String getCountryMaster() {
 	}
 
 	@ApiOperation(value = "Get country city master for beneficiary", consumes = "application/json", produces = "application/json")
-	@RequestMapping(value = "/get/countryCityMaster/{countryID}/", method = RequestMethod.GET)
+	@RequestMapping(value = "/get/countryCityMaster/{countryID}", method = RequestMethod.GET)
```
Comment (roopesh-beehyv) [inline src/main/java/com/iemr/hwc/controller/location/LocationController.java]: Revert this change

File: src/main/java/com/iemr/hwc/controller/location/LocationController.java [hunk 3]
```diff
@@ -148,6 +148,20 @@ public String getVillageMaster(@PathVariable("blockID") Integer blockID) {
 		return response.toString();
 	}
 
+	@ApiOperation(value = "Get village from district id", consumes = "application/json", produces = "application/json")
+	@RequestMapping(value = "/get/villageFromDistrictID/{districtID}", method = RequestMethod.GET)
+	public String getVillageByDistrictID(@PathVariable("districtID") Integer districtID) {
+		logger.info("get village by districtID ..." + districtID);
+		response = new OutputResponse();
+		String s = locationServiceImpl.getVillageListByDistrictID(districtID);
+		if (s != null)
```
Comment (roopesh-beehyv) [inline src/main/java/com/iemr/hwc/controller/location/LocationController.java]: add brackets ( {}) for if and else

File: src/main/resources/application.properties [hunk 4]
```diff
@@ -21,7 +21,7 @@ spring.jpa.hibernate.show_sql=true
 spring.jpa.hibernate.format_sql=true
 
 spring.session.store-type=redis
-#spring.redis.host=localhost
+spring.redis.host=localhost
```
Comment (roopesh-beehyv) [inline src/main/resources/application.properties]: Revert this change

File: src/main/java/com/iemr/hwc/service/fetosense/FetosenseServiceImpl.java [hunk 5]
```diff
@@ -73,6 +73,9 @@ public class FetosenseServiceImpl implements FetosenseService {
 	@Value("${fetosenseAPIKey}")
 	private String fetosenseAPIKey;
 
+	@Value("${fetosenseReportPath}")
+	private String fetosenseReportPath;
+
```
Comment (roopesh-beehyv) [inline src/main/java/com/iemr/hwc/service/fetosense/FetosenseServiceImpl.java:78]: Why this dependency added here?

File: .github/workflows/sast-and-package-prod.yml (hunk 1 above)
Comment (babstar93) [reply .github/workflows/sast-and-package-prod.yml]: This update was done by @Vidyaaa24 on this commit: https://github.com/beehyv/PSMRI-HWC-API/commit/8b71c28c9d1a873b76bae3311fcf25ecfea097b6

File: src/main/java/com/iemr/hwc/service/fetosense/FetosenseServiceImpl.java (hunk 5 above)
Comment (babstar93) [reply src/main/java/com/iemr/hwc/service/fetosense/FetosenseServiceImpl.java:78]: I didn't change this file

File: src/main/java/com/iemr/hwc/controller/location/LocationController.java (hunk 2 above)
Comment (babstar93) [reply src/main/java/com/iemr/hwc/controller/location/LocationController.java]: Updated

File: src/main/resources/application.properties (hunk 4 above)
Comment (babstar93) [reply src/main/resources/application.properties]: Updated
